airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -1138,7 +1138,7 @@ def stop_pipeline(
         if check_interval is None:
             check_interval = 10
 
-        for retries in (2, 1, 0):
+        for retries in (4, 3, 2, 1, 0):
```

Comment:
Small optional suggestion

```suggestion
        for retries in reversed(range(5)):
```